templates/zerver/integrations/discourse.md

```diff
@@ -1,16 +1,27 @@
+# Zulip Discourse integration
+
+Get Discourse notifications in Zulip!
+
+{start_tabs}
+
 1.  {!create-an-incoming-webhook.md!}
 
-    The API keys for "Incoming webhook" bots are limited to only
-    sending messages via webhooks. Thus, this bot type lessens
-    the security risks associated with exposing the bot's API
-    key to third-party services.
+    !!! warn ""
+
+        **Note:** The API keys for "Incoming webhook" bots are limited to
+        only sending messages via webhooks. Thus, this bot type lessens
+        the security risks associated with exposing the bot's API key to
+        third-party services.
 
 1.  Copy the `USERNAME` and `API KEY` - you'll need it later.
 
-1.  Head over to the
-    [discourse-chat-integration setup instructions](https://meta.discourse.org/t/68501)
-    and complete them.
+1.  Complete the
+    [discourse-chat-integration setup instructions][setup-instructions].
```

Comment:
This is really interesting. We usually document these steps in our docs as well. And that documentation on Discourse's site looks a bit out of date, at least the section about creating a bot and the notes about stream/subject. I wonder if there's a way for us to submit an update for their docs?

Anyway, maybe worth a conversation in  CZO about this specific integration documentation as to whether we want to continue to link to that or add those documentation steps to our own.